spark/src/main/scala/geotrellis/spark/ingest/package.scala

```diff
@@ -15,21 +15,25 @@ package object ingest {
   implicit class ProjectedExtentComponentMethods[T: ProjectedExtentComponent](key: T) {
     val _projectedExtent = implicitly[ProjectedExtentComponent[T]]
 
-    def projectedExtent: ProjectedExtent = key &|-> _projectedExtent.lens get
+    def projectedExtent: ProjectedExtent = _projectedExtent.lens.get(key)
 
     def updateProjectedExtent(pe: ProjectedExtent): T =
-      key &|-> _projectedExtent.lens set (pe)
+      _projectedExtent.lens.set(pe)(key)
   }
 
   implicit object ProjectedExtentComponent extends IdentityComponent[ProjectedExtent]
 
-  implicit class withCollectMetadataMethods[K: ProjectedExtentComponent, V <: CellGrid](rdd: RDD[(K, V)]) extends Serializable {
-    def collectMetaData(crs: CRS, layoutScheme: LayoutScheme): (Int, RasterMetaData) = {
-      RasterMetaData.fromRdd(rdd, crs, layoutScheme)(_.projectedExtent.extent)
+  implicit class withCollectMetadataMethods[
```

Comment:
Should we make a `metadata` package that would have these methods? I'm a bit uncomfortable with still putting stuff in `ingest`, which we have talked a bunch about getting rid of.